src/packages/@ncigdc/modern_components/GeneExpression/inchlib/index.js

```diff
@@ -2087,7 +2103,14 @@ import ageDisplay, { getLowerAgeYears } from '@ncigdc/utils/ageDisplay';
       text_value = value;
 
       if (value !== null) {
-        color = self._get_color_for_value(value, self.data_descs[col_index].min, self.data_descs[col_index].max, self.data_descs[col_index].middle, self.options.heatmap_colors);
+        color = self._get_color_for_value('heatmap')(
+          value,
+          self.data_descs[col_index].min,
+          self.data_descs[col_index].max,
+          ['median', 'geneExpression'].includes(self.options.centering)
+            ? 0
```

Comment:
This centres the colour assignment for the heatmap itself.
Used column averaging by default.